tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
```

Comment:
So now, rather than invoking `make test`, we invoke `tox -e test`, right? If so, this is great! What do you think about keeping the `Makefile` around but replacing the inner calls with `tox -e`? 🙂 Curious whether that might be nicer for people who may know `make` but not necessarily `tox`.

Reply:
> So now, rather than invoking make test, we invoke tox -e test, right?

There's no exact counterpart of `make test` in this branch.
`tox` is the task runner on each isolated environment.
If your want to run test for Python 3.8, then you should invoke `tox -e py38`. 
`make lint`, `make format` correspond to `tox -e lint` `tox -e format`

How about making Makefile below?
```Makefile
testall: # run all test in all Python version
	tox

test-pyXX: # run test on PythonXX
	tox -e pyXX
...
```